### PR TITLE
chore: Add pre-commits

### PR DESCRIPTION
**Pull request by saattrupdan**
Adds the following pre-commit hooks:

- `black`: Stylistic changes and linting
- `flake8`: Linting
- `isort`: Ordering of imports

Note that the pre-commit hooks have to be installed using `pre-commit install`, and during the first commit all the hooks will be installed, so it will take a few seconds longer than normal - after that, it's instantaneous.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 22.3.0
+    hooks:
+    - id: black
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+    - id: flake8
```
Comment (KennethEnevoldsen) [inline .pre-commit-config.yaml:9]: I would move flake at the end as it is a check which might be influenced by isort.

**Review by KennethEnevoldsen — APPROVED** [review]
I would change the order of flake8 and isort otherwise everything looks good.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (saattrupdan) [reply .pre-commit-config.yaml]: Sure thing

**Comment by KennethEnevoldsen** [comment]
<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/README.md">Coverage</a><br/><details><summary>Coverage Report</summary><table><tr><th>File</th><th>Stmts</th><th>Miss</th><th>Cover</th><th>Missing</th></tr><tbody><tr><td colspan="5"><b>src/dfm</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/__init__.py">__init__.py</a></td><td>1</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td colspan="5"><b>src/dfm/cleaning</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/__init__.py">__init__.py</a></td><td>2</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py">deduper.py</a></td><td>156</td><td>16</td><td>90%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L119-L124">119&ndash;124</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L147">147</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L275">275</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L278">278</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L311">311</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L406">406</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L421">421</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L444">444</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L549">549</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper.py#L576-L579">576&ndash;579</a></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper_utils.py">deduper_utils.py</a></td><td>26</td><td>3</td><td>88%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/deduper_utils.py#L57-L60">57&ndash;60</a></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py">quality_filter.py</a></td><td>220</td><td>32</td><td>85%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L78-L83">78&ndash;83</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L264">264</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L269">269</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L343">343</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L352">352</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L391-L406">391&ndash;406</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L539">539</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L572-L575">572&ndash;575</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L600">600</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L632">632</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L716">716</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/cleaning/quality_filter.py#L730">730</a></td></tr><tr><td colspan="5"><b>src/dfm/data</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/data/__init__.py">__init__.py</a></td><td>0</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/data/load_datasets.py">load_datasets.py</a></td><td>88</td><td>88</td><td>0%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/data/load_datasets.py#L5-L344">5&ndash;344</a></td></tr><tr><td colspan="5"><b>src/dfm/description</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/__init__.py">__init__.py</a></td><td>1</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/description_patterns.py">description_patterns.py</a></td><td>48</td><td>32</td><td>33%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/description_patterns.py#L500">500</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/description_patterns.py#L541">541</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/description_patterns.py#L581">581</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/description_patterns.py#L764-L769">764&ndash;769</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/description_patterns.py#L780-L793">780&ndash;793</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/description_patterns.py#L802-L819">802&ndash;819</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/description_patterns.py#L828-L845">828&ndash;845</a></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/generate_description.py">generate_description.py</a></td><td>12</td><td>12</td><td>0%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/generate_description.py#L1-L39">1&ndash;39</a></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/description/match_counter.py">match_counter.py</a></td><td>52</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td colspan="5"><b>src/dfm/dfm_tokenizers</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/dfm_tokenizers/__init__.py">__init__.py</a></td><td>2</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/dfm_tokenizers/tokenizer_config.py">tokenizer_config.py</a></td><td>34</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/dfm_tokenizers/train_tokenizer.py">train_tokenizer.py</a></td><td>67</td><td>5</td><td>93%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/dfm_tokenizers/train_tokenizer.py#L50">50</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/dfm_tokenizers/train_tokenizer.py#L118">118</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/dfm_tokenizers/train_tokenizer.py#L120">120</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/f93221e200232d8d927f6f5d17561a8aea174ac3/src/dfm/dfm_tokenizers/train_tokenizer.py#L160-L161">160&ndash;161</a></td></tr><tr><td><b>TOTAL</b></td><td><b>709</b></td><td><b>188</b></td><td><b>73%</b></td><td>&nbsp;</td></tr></tbody></table></details>

**Comment by KennethEnevoldsen** [comment]
<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/README.md">Coverage</a><br/><details><summary>Coverage Report</summary><table><tr><th>File</th><th>Stmts</th><th>Miss</th><th>Cover</th><th>Missing</th></tr><tbody><tr><td colspan="5"><b>src/dfm</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/__init__.py">__init__.py</a></td><td>1</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td colspan="5"><b>src/dfm/cleaning</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/__init__.py">__init__.py</a></td><td>2</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py">deduper.py</a></td><td>156</td><td>16</td><td>90%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L119-L124">119&ndash;124</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L147">147</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L275">275</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L278">278</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L311">311</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L406">406</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L421">421</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L444">444</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L549">549</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper.py#L576-L579">576&ndash;579</a></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper_utils.py">deduper_utils.py</a></td><td>26</td><td>3</td><td>88%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/deduper_utils.py#L57-L60">57&ndash;60</a></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py">quality_filter.py</a></td><td>220</td><td>32</td><td>85%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L78-L83">78&ndash;83</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L264">264</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L269">269</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L343">343</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L352">352</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L391-L406">391&ndash;406</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L539">539</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L572-L575">572&ndash;575</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L600">600</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L632">632</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L716">716</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/cleaning/quality_filter.py#L730">730</a></td></tr><tr><td colspan="5"><b>src/dfm/data</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/data/__init__.py">__init__.py</a></td><td>0</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/data/load_datasets.py">load_datasets.py</a></td><td>88</td><td>88</td><td>0%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/data/load_datasets.py#L5-L344">5&ndash;344</a></td></tr><tr><td colspan="5"><b>src/dfm/description</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/__init__.py">__init__.py</a></td><td>1</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/description_patterns.py">description_patterns.py</a></td><td>48</td><td>32</td><td>33%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/description_patterns.py#L500">500</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/description_patterns.py#L541">541</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/description_patterns.py#L581">581</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/description_patterns.py#L764-L769">764&ndash;769</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/description_patterns.py#L780-L793">780&ndash;793</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/description_patterns.py#L802-L819">802&ndash;819</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/description_patterns.py#L828-L845">828&ndash;845</a></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/generate_description.py">generate_description.py</a></td><td>12</td><td>12</td><td>0%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/generate_description.py#L1-L39">1&ndash;39</a></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/description/match_counter.py">match_counter.py</a></td><td>52</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td colspan="5"><b>src/dfm/dfm_tokenizers</b></td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/dfm_tokenizers/__init__.py">__init__.py</a></td><td>2</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/dfm_tokenizers/tokenizer_config.py">tokenizer_config.py</a></td><td>34</td><td>0</td><td>100%</td><td>&nbsp;</td></tr><tr><td>&nbsp; &nbsp;<a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/dfm_tokenizers/train_tokenizer.py">train_tokenizer.py</a></td><td>67</td><td>5</td><td>93%</td><td><a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/dfm_tokenizers/train_tokenizer.py#L50">50</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/dfm_tokenizers/train_tokenizer.py#L118">118</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/dfm_tokenizers/train_tokenizer.py#L120">120</a>, <a href="https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/214566c03128e729d3193e48b75c396d7d4c5575/src/dfm/dfm_tokenizers/train_tokenizer.py#L160-L161">160&ndash;161</a></td></tr><tr><td><b>TOTAL</b></td><td><b>709</b></td><td><b>188</b></td><td><b>73%</b></td><td>&nbsp;</td></tr></tbody></table></details>

**Pull request merged**